ui/tests/integration/components/form-field-label-test.js

```diff
@@ -44,8 +44,6 @@ module('Integration | Component | form-field-label', function (hooks) {
     assert.dom('a').doesNotExist('docLink hidden when not provided');
     this.set('docLink', '/doc/path');
     assert.dom('.sub-text').includesText('See our documentation for help', 'Doc link text renders');
-    assert
-      .dom('a')
-      .hasAttribute('href', 'https://developer.hashicorp.com' + this.docLink, 'Doc link renders');
+    assert.dom('a').hasAttribute('href', 'https://openbao.org' + this.docLink, 'Doc link renders');
```

Comment:
Do you think it is worthwhile re-enabling the UI tests now?

Reply:
I don't think so, some of them still fail, or maybe in a non-blocking way